sdk/hybridconnectivity/Azure.ResourceManager.HybridConnectivity/src/Generated/Models/GenerateAwsTemplateContent.cs

```diff
@@ -0,0 +1,80 @@
+        /// <para>
+        /// To assign an object to the value of this property use <see cref="BinaryData.FromObjectAsJson{T}(T, System.Text.Json.JsonSerializerOptions?)"/>.
+        /// </para>
+        /// <para>
+        /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
+        /// </para>
+        /// <para>
+        /// Examples:
+        /// <list type="bullet">
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson("foo")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("\"foo\"")</term>
+        /// <description>Creates a payload of "foo".</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromObjectAsJson(new { key = "value" })</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// <item>
+        /// <term>BinaryData.FromString("{\"key\": \"value\"}")</term>
+        /// <description>Creates a payload of { "key": "value" }.</description>
+        /// </item>
+        /// </list>
+        /// </para>
+        /// </summary>
+        private IDictionary<string, BinaryData> _serializedAdditionalRawData;
+
+        /// <summary> Initializes a new instance of <see cref="GenerateAwsTemplateContent"/>. </summary>
+        /// <param name="connectorId"> The name of public cloud connector. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="connectorId"/> is null. </exception>
+        public GenerateAwsTemplateContent(string connectorId)
+        {
+            Argument.AssertNotNull(connectorId, nameof(connectorId));
+
+            ConnectorId = connectorId;
+            SolutionTypes = new ChangeTrackingList<SolutionTypeSettings>();
+        }
+
+        /// <summary> Initializes a new instance of <see cref="GenerateAwsTemplateContent"/>. </summary>
+        /// <param name="connectorId"> The name of public cloud connector. </param>
+        /// <param name="solutionTypes"> The list of solution types and their settings. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal GenerateAwsTemplateContent(string connectorId, IList<SolutionTypeSettings> solutionTypes, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            ConnectorId = connectorId;
+            SolutionTypes = solutionTypes;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="GenerateAwsTemplateContent"/> for deserialization. </summary>
+        internal GenerateAwsTemplateContent()
+        {
+        }
+
+        /// <summary> The name of public cloud connector. </summary>
+        public string ConnectorId { get; }
```

Comment:
Is it a GUID?

Reply:
It's arm-id, transformed to resource identifier